apis/base.go

```diff
@@ -92,6 +92,7 @@ func InitApi(app core.App) (*echo.Echo, error) {
 
 	// default routes
 	api := e.Group("/api")
+	defer bindHealthApi(app, api) // health check should always be initialized after everything else is done
```

Comment:
Was there any particular reason why the `defer` is needed?
It should work just fine without it.

Reply:
My thought here was to delay the health check to the end, so that it's only available if everything else passed without an error, but I can change it if you want.

Reply:
Let's remove it.

The `defer` will call the bind even if the InitApi returns an error so it shouldn't matter in this case. Additionally it is better the internal routes to be registered before the `OnBeforeServe` hook trigger so that it can return an error to the user in case they decide to register a new endpoint with the same path.

In future, we may update the healthcheck handler so that it actually checks whether the APP failed to initialize or is still in the process, but for now let's keep it simple.

Reply:
Done